terraform/modules/alertmanager/config/slack.tmpl

```diff
@@ -0,0 +1,86 @@
+# This builds the silence URL.  We exclude the alertname in the range
```

Comment:
As this was copied from an existing file, can you add a link to the original and credit them?